doc/api/cli.md

```diff
@@ -121,6 +121,13 @@ added: v6.0.0
 
 Print stack traces for process warnings (including deprecations).
 
+### `--redirect-warnings=file`
```

Comment:
Also, docs don't say whether file is created or not, or whether it is appended to, or replaces the file.